### PR TITLE
Added: humanification/simulate typing

### PR DESCRIPTION
**Pull request by ujja**

### CODE REVIEW

**Comment by codeforequity-at** [comment]
I am seeing a conflict in Convo.js - please do a forward integration from master to the feature branch

File: src/scripting/Convo.js [hunk 1]
```diff
@@ -62,7 +62,15 @@ class Convo {
           if (convoStep.sender === 'me') {
             convoStep.messageText = this._checkNormalizeText(container, scriptingMemory, convoStep.messageText)
             debug(`${this.header.name}/${convoStep.stepTag}: user says ${JSON.stringify(convoStep, null, 2)}`)
-            container.UserSays(new BotiumMockMessage(convoStep))
+
+            new Promise(resolve => {
+              if (container.caps.SIMULATE_WRITING_SPEED) {
+                setTimeout(() => resolve(), container.caps.SIMULATE_WRITING_SPEED * convoStep.messageText.length)
```
Comment (codeforequity-at) [inline src/scripting/Convo.js]: messageText could be missing (for click&point bot interfaces ...). please consider this here.

**Comment by ujja** [comment]
> I am seeing a conflict in Convo.js - please do a forward integration from master to the feature branch

done

File: src/scripting/Convo.js (hunk 1 above)
Comment (ujja) [reply src/scripting/Convo.js]: done

**Pull request merged**